### ISSUE TITLE
randomize() broken in JS mode.

### ISSUE THREAD

**Issue by treeform** [nim-lang/Nim#11772]
I think JS started to fail with randomize() because the int seed is too big.
I think the solution would to to make a smaller seed that JS can handle.
I have a solution where I grab the lower parts of the epoch time as the seed.

### Example
```nim
import random
randomize()
```

### Current Output
```
> nim js -r bug.nim
/p/bug.js:462
    throw new Error(cbuf_64601);
    ^

Error: Error: unhandled exception: value out of range [RangeError]
Traceback (most recent call last)
random.randomize, line: 612
random.randomize, line: 559
random.initRand, line: 539

    at unhandledException (/p/bug.js:462:11)
    at raiseException (/p/bug.js:251:3)
    at raiseRangeError (/p/bug.js:513:3)
    at chckRange (/p/bug.js:271:3)
    at init_rand_89397 (/p/bug.js:565:24)
    at randomize_89622 (/p/bug.js:580:27)
    at randomize_222001 (/p/bug.js:592:3)
    at Object.<anonymous> (/p/bug.js:597:1)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
Error: execution of an external program failed: '/usr/local/Cellar/node/8.0.0_1/bin/node /p/bug.js '
```

### Possible Solution
```nim
randomize((epochTime()*1000).int and 0xFFFFFFFF)
```

```
$ nim -v 
Nim Compiler Version 0.20.99 [MacOSX: amd64]
Compiled at 2019-07-16
Copyright (c) 2006-2019 by Andreas Rumpf

active boot switches: -d:release
```

**Comment by treeform** [comment]
Opps I just saw this: https://github.com/nim-lang/Nim/issues/11450

trying out with todays version.

**Comment by treeform** [comment]
Just did `choosenim update devel` - still broken.

This appears to be a very related but a different issue.

**Comment by treeform** [comment]
@narimiran could you look into this as you did the other fix on #11450 ?

**Comment by narimiran** [comment]
> could you look into this as you did the other fix on #11450 ?

From what I can tell, the same/similar fix is needed. I've changed `let time = int64(times.epochTime() * 1_000_000_000)` to `let time = int64(times.epochTime() * 100_000)`, but this is now still too large.
